src/utils/helper-functions.tsx

```diff
@@ -114,3 +114,34 @@ export const spliceSx = (defaultStyles?: SxProps, newStyles?: SxProps) =>
 
 export const studentOrTutor = (userRole: '' | UserRole) =>
   userRole === UserRoleEnum.Tutor ? UserRoleEnum.Tutor : UserRoleEnum.Student
+
+export const getDifferenceDates = (
+  startDate: string | number,
+  endDate: string | number
+) => {
+  const difference: number = new Date(endDate) - new Date(startDate)
```

Comment:
it should has type Date as i see. not a number

Reply:
No, you will receive the value type of number.

Reply:
It's date (startDate: Date, endDate: Date)